packages/fxa-payments-server/src/components/PaymentForm/index.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useRef } from 'react';
+import { render, cleanup, prettyDOM } from '@testing-library/react';
+import 'jest-dom/extend-expect';
+import { StripeProvider } from 'react-stripe-elements';
+import PaymentForm, { PaymentFormProps } from './index';
+
+let elementMock;
+let elementsMock: { create: jest.Mock<any,any> };
+let stripeMock: stripe.Stripe;
+
+beforeEach(() => {
+  elementMock = {
+    mount: jest.fn(),
+    destroy: jest.fn(),
+    on: jest.fn(),
+    update: jest.fn(),
+  };
+  elementsMock = {
+    create: jest.fn().mockReturnValue(elementMock),
+  };
+  stripeMock = {
+    elements: jest.fn().mockReturnValue(elementsMock),
+    createToken: jest.fn(),
+    createSource: jest.fn(),
+    createPaymentMethod: jest.fn(),
+    handleCardPayment: jest.fn(),
+    retrieveSource: jest.fn(),
+    paymentRequest:jest.fn(),
+    redirectToCheckout: jest.fn(),
+    retrievePaymentIntent: jest.fn(),
+    handleCardAction: jest.fn(),
+    confirmPaymentIntent: jest.fn(),
+  };
+});
+
+afterEach(cleanup);
+
+type SubjectProps = PaymentFormProps;
+
+const Subject = (props: SubjectProps) => {
+  return (
+    <StripeProvider stripe={stripeMock}>
+      <PaymentForm {...props} />
+    </StripeProvider>
+  );
+};
+
+it('renders all expected fields', () => {
```

Comment:
Ugh. Still kind of a TODO here: Couldn't quite figure out how to get the third-party JS include for Stripe components to play nicely with unit tests here. So, I punted on this particular test suite to at least assert that the right fields are being rendered in the form. On the bright side, I have tests for almost everything else surrounding these fields except for the actual Stripe components